checks/raw/binary_artifact.go

```diff
@@ -89,9 +90,10 @@ func excludeValidatedGradleWrappers(c clients.RepoClient, files []checker.File)
 	// Remove Gradle wrapper JARs from files.
 	filterFiles := []checker.File{}
 	for _, f := range files {
-		if filepath.Base(f.Path) != "gradle-wrapper.jar" {
-			filterFiles = append(filterFiles, f)
+		if filepath.Base(f.Path) == "gradle-wrapper.jar" {
+			f.Type = finding.FileTypeGradleWrapper
 		}
+		filterFiles = append(filterFiles, f)
```

Comment:
We're not filtering anything anymore, just labeling. can we just loop in place and modify the type?

```go
for i := range files {
		if filepath.Base(files[i].Path) == "gradle-wrapper.jar" {
			files[i].Type = finding.FileTypeGradleWrapper
		}
```

Reply:
https://github.com/ossf/scorecard/pull/3508/commits/7cb09fcc6543fdebc8ea57e337aa79c0dd326cc6